prism/src/main/java/org/apache/falcon/service/EntitySLAListener.java

```diff
@@ -18,14 +18,13 @@
 package org.apache.falcon.service;
 
 import org.apache.falcon.FalconException;
-import org.apache.falcon.entity.v0.EntityType;
 
 import java.util.Date;
 
 /**
  * Interface for FeedSLAAlert to be used by Listeners.
  */
 public interface EntitySLAListener {
-    void highSLAMissed(String enityName, EntityType entityType, String clusterName, Date nominalTime)
+    void highSLAMissed(String entityName, String clusterName, String entityType, Date nominalTime)
```

Comment:
I understand your need to keep entityType a string in the beans. But, the interface should use the enum.